Provider Provisioner/tsconfig.build.json

```diff
@@ -4,7 +4,9 @@
         "outDir": "./dist",
         "rootDir": "src",
         "composite": true,
-        "tsBuildInfoFile": "./dist/tsconfig.build.tsbuildinfo"
+        "tsBuildInfoFile": "./dist/tsconfig.build.tsbuildinfo",
+        "skipLibCheck": true,
+        "noImplicitAny": false
```

Comment:
Setting noImplicitAny to false helped eliminate errors caused by missing @types/lib libraries.